HuntTheWumpus/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.IO;
+
+namespace HuntTheWumpus
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            DisplayRules();
+            DisplayMap();
+        }
+
+        static void DisplayRules()
+        {
+            string rules = File.ReadAllText(@"SupportFiles\wumpus.txt");
```

Comment:
For future release we should handle when file does not exist using something like a try-catch rather than it crashing.

Reply:
Good point!